### ISSUE TITLE
Self contained build with async await

### ISSUE THREAD

**Issue by 4lexBaum** [SAP/ui5-tooling#69]
I'm trying to build a PWA app with UI5 and therefore I have a 'service-worker.js' file with some async / await function like in this [example](https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent#Examples).

## Expected Behavior
Build run successful.

## Current Behavior
Build is failing with 'Unexpected token function'.

## Steps to reproduce the issue
1. Include a .js file in the index.html
2. Use some async functions
3. Run self contained build

## Context
- OS/Platform: MacOS
- Node.js Version: v8.5.0
- npm Version: 5.3.0

## Affected components
- [x] [ui5-builder](https://github.com/SAP/ui5-builder)
- [ ] [ui5-server](https://github.com/SAP/ui5-server)
- [ ] [ui5-cli](https://github.com/SAP/ui5-cli)
- [ ] [ui5-fs](https://github.com/SAP/ui5-fs)
- [ ] [ui5-project](https://github.com/SAP/ui5-project)
- [ ] [ui5-logger](https://github.com/SAP/ui5-logger)

## Stack trace/log output
```
info builder:builder application openui5-sample-app 🔨 (4/8) Running task generateStandaloneAppBundle...
ERR! lbt:resources:ResourcePool Error: Line 56: Unexpected token function
ERR! lbt:resources:ResourcePool     at constructError (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2407:21)
ERR! lbt:resources:ResourcePool     at createError (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2426:17)
ERR! lbt:resources:ResourcePool     at unexpectedTokenError (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2500:13)
ERR! lbt:resources:ResourcePool     at throwUnexpectedToken (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2505:15)
ERR! lbt:resources:ResourcePool     at expect (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2523:13)
ERR! lbt:resources:ResourcePool     at expectCommaSeparator (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2547:13)
ERR! lbt:resources:ResourcePool     at parseArguments (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:3333:17)
ERR! lbt:resources:ResourcePool     at parseLeftHandSideExpressionAllowCall (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:3426:24)
ERR! lbt:resources:ResourcePool     at inheritCoverGrammar (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2681:18)
ERR! lbt:resources:ResourcePool     at parsePostfixExpression (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:3490:16)
ERR! lbt:resources:ResourcePool  failed to parse or analyze %s: sap/ui/demo/todo/service-worker.js { Error: Line 56: Unexpected token function
ERR! lbt:resources:ResourcePool     at constructError (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2407:21)
ERR! lbt:resources:ResourcePool     at createError (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2426:17)
ERR! lbt:resources:ResourcePool     at unexpectedTokenError (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2500:13)
ERR! lbt:resources:ResourcePool     at throwUnexpectedToken (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2505:15)
ERR! lbt:resources:ResourcePool     at expect (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2523:13)
ERR! lbt:resources:ResourcePool     at expectCommaSeparator (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2547:13)
ERR! lbt:resources:ResourcePool     at parseArguments (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:3333:17)
ERR! lbt:resources:ResourcePool     at parseLeftHandSideExpressionAllowCall (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:3426:24)
ERR! lbt:resources:ResourcePool     at inheritCoverGrammar (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2681:18)
ERR! lbt:resources:ResourcePool     at parsePostfixExpression (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:3490:16)
ERR! lbt:resources:ResourcePool   lineNumber: 56,
ERR! lbt:resources:ResourcePool   description: 'Unexpected token function',
ERR! lbt:resources:ResourcePool   index: 1961,
ERR! lbt:resources:ResourcePool   stack: 'Error: Line 56: Unexpected token function\n    at constructError (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2407:21)\n    at createError (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2426:17)\n    at unexpectedTokenError (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2500:13)\n    at throwUnexpectedToken (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2505:15)\n    at expect (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2523:13)\n    at expectCommaSeparator (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2547:13)\n    at parseArguments (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:3333:17)\n    at parseLeftHandSideExpressionAllowCall (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:3426:24)\n    at inheritCoverGrammar (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:2681:18)\n    at parsePostfixExpression (/Users/d062273/.nvm/versions/node/v8.5.0/lib/node_modules/@ui5/cli/node_modules/esprima/esprima.js:3490:16)' }
ERR! lbt:bundle:Resolver **** error: missing module sap/ui/support/Bootstrap.js, required by sap/ui/core/Core.js
```

**Comment by matz3** [comment]
Will be solved with upgrading to esprima v4, see https://github.com/SAP/ui5-builder/pull/64.

**Comment by RandomByte** [comment]
@matz3 has this been resolved with v1.0.0?

**Comment by matz3** [comment]
Yes, can be closed now 👍
